cumulus/parachains/runtimes/collectives/collectives-westend/src/secretary/mod.rs

```diff
@@ -0,0 +1,232 @@
+	type WeightInfo = (); // TODO weights::pallet_core_fellowship_secretary_core::WeightInfo<Runtime>;
+	type RuntimeEvent = RuntimeEvent;
+	type Members = pallet_ranked_collective::Pallet<Runtime, SecretaryCollectiveInstance>;
+	type Balance = Balance;
+	type ParamsOrigin = OpenGovOrFellow;
+	// Induction is by any of:
+	// - Root;
+	// - FellowshipAdmin (i.e. token holder referendum);
+	// - A single Member of the Technical Fellowship, rank 3 and above;
+	// - A single member of the Secretary Collective.
+	type InductOrigin = EitherOfDiverse<
+		EitherOfDiverse<
+			EnsureRoot<AccountId>,
+			EnsureXcm<IsVoiceOfBody<GovernanceLocation, FellowshipAdminBodyId>>,
+		>,
+		EitherOfDiverse<
+			pallet_ranked_collective::EnsureMember<
+				Runtime,
+				FellowshipCollectiveInstance,
+				{ DAN_3 },
+			>,
+			pallet_ranked_collective::EnsureMember<
+				Runtime,
+				SecretaryCollectiveInstance,
+				{ ranks::SECRETARY },
+			>,
+		>,
+	>;
+	type ApproveOrigin = ApproveOrigin;
+	type PromoteOrigin = ApproveOrigin;
+	type EvidenceSize = ConstU32<65536>;
+	type MaxRank = ConstU32<1>;
+}
+
+pub type SecretarySalaryInstance = pallet_salary::Instance3;
+
+parameter_types! {
+	// The interior location on AssetHub for the paying account. This is the Secretary Salary
+	// pallet instance. This sovereign account will need funding.
+	pub SecretarySalaryInteriorLocation: InteriorLocation = PalletInstance(94).into();
+}
+
+/// [`PayOverXcm`] setup to pay the Secretary salary on the AssetHub in USDT.
+pub type SecretarySalaryPaymaster = PayOverXcm<
+	SecretarySalaryInteriorLocation,
+	crate::xcm_config::XcmRouter,
+	crate::PolkadotXcm,
+	ConstU32<{ 6 * HOURS }>,
+	AccountId,
+	(),
+	ConvertToValue<WndAssetHub>,
+	AliasesIntoAccountId32<(), AccountId>,
+>;
+
+pub struct SalaryForRank;
+impl GetSalary<u16, AccountId, Balance> for SalaryForRank {
+	fn get_salary(rank: u16, _who: &AccountId) -> Balance {
+		if rank == 1 {
+			1000
+		} else {
+			0
+		}
+	}
+}
+
+impl pallet_salary::Config<SecretarySalaryInstance> for Runtime {
+	type WeightInfo = (); // TODO weights::pallet_salary_secretary_salary::WeightInfo<Runtime>;
+	type RuntimeEvent = RuntimeEvent;
+
+	#[cfg(not(feature = "runtime-benchmarks"))]
+	type Paymaster = SecretarySalaryPaymaster;
+	#[cfg(feature = "runtime-benchmarks")]
+	type Paymaster = crate::impls::benchmarks::PayWithEnsure<
+		SecretarySalaryPaymaster,
+		crate::impls::benchmarks::OpenHrmpChannel<ConstU32<1000>>,
+	>;
+	type Members = pallet_ranked_collective::Pallet<Runtime, SecretaryCollectiveInstance>;
+
+	#[cfg(not(feature = "runtime-benchmarks"))]
+	type Salary = SalaryForRank;
+	#[cfg(feature = "runtime-benchmarks")]
+	type Salary = frame_support::traits::tokens::ConvertRank<
+		crate::impls::benchmarks::RankToSalary<Balances>,
+	>;
+	// 15 days to register for a salary payment.
+	type RegistrationPeriod = ConstU32<{ 15 * DAYS }>;
+	// 15 days to claim the salary payment.
+	type PayoutPeriod = ConstU32<{ 15 * DAYS }>;
```

Comment:
```suggestion
	type RegistrationPeriod = ConstU32<{ 6 * HOURS }>;
```
Should reduce these for faster testing.